### DISCUSSION TITLE
Error in Analysing the data for ABFE

### DISCUSSION THREAD

**Discussion by jjmallen** [alchemistry/alchemlyb#392] [Q&A]
![Screenshot 2024-09-04 at 20 49 51](https://github.com/user-attachments/assets/156de995-2ef3-41ee-8a63-db9d5ce7e0f2)
Good day everyone

I have completed my ABFE calculation with gromacs and i have obtained my .xvg files. When I am trying to analyse the results using alchemlyb but I am getting the error attached in the picture. Kindly assist on how I can fix this error

**Comment by xiki-tempula** [comment]
Do you mind share the free energy data, please? Thanks.

**Reply by jjmallen** [reply]
[complex.zip](https://github.com/user-attachments/files/16895513/complex.zip)
[ligand.zip](https://github.com/user-attachments/files/16895514/ligand.zip)
these are the folders with dhdl files

**Reply by jjmallen** [reply]
Good day sir, 
any update with the error I am having? kindly assist

**Reply by xiki-tempula** [reply]
I think there are some xvg files missing from the zip file, where I could only find 23 files out of the 30 lambdas.

**Reply by jjmallen** [reply]
Good day, 

Sorry for taking so long to get back to you. I was fixing the simulation so that I could have all the xvg files. please find below the zip file for the files
[complex.zip](https://github.com/user-attachments/files/17050123/complex.zip)

**Reply by jjmallen** [reply]
and i am still getting the same error even with all the xvg files

**Reply by xiki-tempula** [reply]
I did
```
from alchemlyb.workflows import ABFE
wf = ABFE(dir='.', T=298)
wf.run()
```
This seems to run fine. Do you mind give it alchemlyb an update and see if it solves the problem?
